NSS-Spontaneous-Spoonbills/Sprint2
# Js employee list

## Link to Ticket

Autolink format: hashtag and number, ex: #1094

Closes #
No ticket closed. Tests needed to be completed prior to ticket issue 1 being closed.

## Description of Proposed Changes
Created models, views, & templates that upon clicking on employees listed the the employees name and department.

## Steps to Test
1. Run virtual environment
1. Run server
1. Create a department, computer, & employee
1. Delete migrations & db.sqlite file
1. Don't add an input in you DB Browser when adding the start_date
1. Make sure migrations are created 

## Outline the steps to test
1. Once your environment is up and running, click on the employees nav button
1. Check and see that the list of all employees and their department are created

```sh
git fetch --all
git checkout js_employee_list
```

## Impacted Areas in Application

##### Modified

	WorkforceManagement/models/__init__.py
	WorkforceManagement/templates/WorkforceManagement/index.html
	WorkforceManagement/urls.py
	WorkforceManagement/models/Department_Model.py
	WorkforceManagement/models/Employee_Model.py
        WorkforceManagement/views/Employee_View.py

##### Added

	WorkforceManagement/templates/WorkforceManagement/Employee_List.html

##### Deleted

	WorkforceManagement/templates/WorkforceManagement/employees_list.html


#### List general components of the application that this PR will affect:

## Mentions @username
@obscure-shadow @binauralbeat @MrErin @kimberly-bird 
Tag users that need to review this code

## Testing
No unit tests have been created for this feature. Next branch will have completed tests



## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
